Weather Alert/CoreData/City+CoreDataProperties.swift

```diff
@@ -6,22 +6,65 @@
 //  Copyright © 2016 UKFast. All rights reserved.
 //
 
-import Foundation
 import CoreData
+import UIKit
 
 
 extension City {
 
     @nonobjc public class func fetchRequest() -> NSFetchRequest<City> {
         return NSFetchRequest<City>(entityName: "City");
     }
-
+    
+    @NSManaged public var name: String?
     @NSManaged public var country: String?
-    @NSManaged public var deg: NSNumber?
     @NSManaged public var id: NSNumber?
     @NSManaged public var lat: NSNumber?
     @NSManaged public var lon: NSNumber?
-    @NSManaged public var name: String?
+    @NSManaged public var deg: NSNumber?
     @NSManaged public var speed: NSNumber?
+    @NSManaged public var favourite: NSNumber?
+    
+}
 
+extension City {
+    
+    
+    var direction: String {
+
+        var direction: String = "Unknown"
+        
+        if let direct: Int = deg as? Int {
+            
+            if direct > 315 || direct <= 45 {
```

Comment:
Consider using an enum instead of hard coded value at this point for readability purposes.
The more things you use, the better you'll understand when to and how to use it.